image-spec.csv

```diff
@@ -1,8 +1,9 @@
 context_dir,image_tag,from_tag
+./base/ubi/9.4-minimal,base-ubi-9.4-minimal
 ./base/ubuntu/22.04,base-ubuntu-22.04,
```

Comment:
i don't think we need to build it anymore, given this switches the dependent images as well